cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/writer/BaloFileWriter.java

```diff
@@ -310,6 +312,11 @@ private void addPlatformLibs(Path root, Path projectDirectory, String moduleName
                     .map(lib -> Paths.get(lib.getPath())).collect(Collectors.toList());
 
             for (Path lib : libs) {
+                if (os.contains("win")) {
```

Comment:
Can we move this logic in to the model instead of having it at the balo writer.

Reply:
yes, moved to model